.rubocop.yml

```diff
@@ -22,4 +22,63 @@ Metrics/BlockLength:
   Exclude:
     - 'Rakefile'
     - '**/*.rake'
-    - 'spec/**/*.rb'
\ No newline at end of file
+    - 'spec/**/*.rb'
+
+#Note: Beginning of old .rubocop_todo.yml below this line.
+
+# Offense count: 1
+Metrics/AbcSize:
+  Max: 22
+
+# Offense count: 1
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
+  Max: 37
+
+# Offense count: 70
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 154
+
+# Offense count: 2
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 15
+
+# Offense count: 1
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: nested, compact
+Style/ClassAndModuleChildren:
+  Exclude:
+    - 'app/controllers/users/omniauth_callbacks_controller.rb'
+
+# Offense count: 13
+Style/Documentation:

```

Comment:
Instead of excluding everything (because we will have to do it for future files as well), let's disable this check.